api_tests/utils.py

```diff
@@ -25,3 +25,6 @@ def urlparse_drop_netloc(url):
     if url[4]:
         return url[2] + '?' + url[4]
     return url[2]
+
+def assert_dict_contains_subset(a, b):
+    assert set(b.keys()) >= a.keys() and {k: b[k] for k in a if k in b} == a
```

Comment:
A slightly more straightforward implementation was posted on StackOverflow: https://stackoverflow.com/questions/21058230/how-to-assert-a-dict-contains-another-dict-without-assertdictcontainssubset-in-p